tests/testthat/test-climatic_summary.R

```diff
@@ -1,43 +1,94 @@
 library(cdms.products)
+library(dplyr)
+library(naflex)
 
-x_daily <- daily_niger %>% dplyr::group_by(date) %>% dplyr::summarize(mean_rain = mean(rain), st_dv_rain = sd(rain), n_na_rain = naflex::na_n(rain))
-x_monthly <- daily_niger %>% dplyr::group_by(month) %>% dplyr::summarize(mean_rain = mean(rain), st_dv_rain = sd(rain), n_na_rain = naflex::na_n(rain))
-x_annual <- daily_niger %>% dplyr::group_by(year) %>% dplyr::summarize(mean_rain = mean(rain), st_dv_rain = sd(rain), n_na_rain = naflex::na_n(rain))
-x_station <- daily_niger %>% dplyr::group_by(station_name) %>% dplyr::summarize(mean_rain = mean(rain), st_dv_rain = sd(rain), n_na_rain = naflex::na_n(rain))
+niger <- daily_niger %>%
+  filter(year %in% 1950:1951)
```

Comment:
@HawardKetoyoMsatsi I also restricted the data to two years to reduce the time it takes to run tests. This should still give enough data to check correctness.